api/v1alpha1/runner_types.go

```diff
@@ -21,25 +21,52 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	KeyRunnerName       = "actions.summerwind.dev/runner-name"
+	KeyRunnerRepository = "actions.summerwind.dev/runner-repository"
+
+	EnvRunnerName       = "RUNNER_NAME"
+	EnvRunnerRepository = "RUNNER_REPO"
+	EnvRunnerToken      = "RUNNER_TOKEN"
+
+	ContainerName = "runner"
+)
+
 // RunnerSpec defines the desired state of Runner
 type RunnerSpec struct {
 	// +kubebuilder:validation:MinLength=3
 	// +kubebuilder:validation:Pattern=`^[^/]+/[^/]+$`
 	Repository string `json:"repository"`
 
 	// +optional
-	Image string `json:"image"`
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
Could you use `RunnerSet` or something similar for managing a stateful sets instead of pods?
Doing so will allow me to deploy both pod-based and stateful-set based runners for comparison against real workloads.

// If that makes sense, I'll rename the current `RunnnerSet` that I've added in #1 to `RunnerReplicaSet` to not collide and make mine look like a less recommended way.

Reply:
Just submitted #6 for the rename

Reply:
Thank you for your helpful comment and PR! Using the `RunnerSet` resource for StatefulSet based management is a great idea. I'm going to merge #6 after reviewing.